client/src/components/molecules/DonationItemForm.tsx

```diff
@@ -0,0 +1,307 @@
+                                />
+                            }
+                            dropdownItems={Array.from(ItemAgeToDescriptionMap.entries()).map(
+                                ([ageKey, description]) =>
+                                    ageKey > 0 && (
+                                        <span key={ageKey} onClick={() => onAgeChange(ageKey)}>
+                                            {description}
+                                        </span>
+                                    )
+                            )}
+                            onDropdownClose={() => {
+                                setIsAgeDropdownOpen(false);
+                            }}
+                            onDropdownOpen={() => {
+                                setIsAgeDropdownOpen(true);
+                            }}
+                            isDropdownOpened={isAgeDropdownOpen}
+                        />
+                    }
+                />
+                <FormItem
+                    className="item-quantity-field"
+                    formItemName="Quantity *"
+                    errorString=""
+                    isDisabled={false}
+                    showErrorIcon={false}
+                    tooltipText="Please input the number of identical items you are requesting to donate"
+                    inputComponent={
+                        <TextField
+                            input={quantity.toString()}
+                            isDisabled={false}
+                            isErroneous={isQuantityErrorneous}
+                            onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                                onQuantityChange(parseInt(e.target.value));
+                            }}
+                            name="quantity"
+                            placeholder=""
+                            type="number"
+                            showRedErrorText={true}
+                        />
+                    }
+                />
+                <FormItem
+                    className="item-description-field"
+                    formItemName="Item Description"
+                    errorString=""
+                    isDisabled={false}
+                    showErrorIcon={false}
+                    inputComponent={
+                        <TextArea
+                            isErroneous={false}
+                            onChange={(e: React.ChangeEvent<HTMLTextAreaElement>) => {
+                                onDescriptionChange(e.target.value);
+                            }}
+                            placeholder="Tell us a little bit about your item. E.x. the bottle is plastic and is 500mL"
+                            value={description}
+                        />
+                    }
+                />
+                <div className="form-buttons">
+                    <Button className="delete-button" text="Delete" copyText="" onClick={props.onDelete} />
```

Comment:
If there is only one item, the delete button shouldn't be visible as per the design

Reply:
Good catch! Ended up fixing it in a different PR (See #191)